### ISSUE TITLE
owncloudcmd bails out with "Authentification failed" after initial success

### ISSUE THREAD

**Issue by felicitus** [owncloud/client#2837]
I'm using owncloud 8.0.0 with 3 clients. Now I wanted to set up owncloudcmd to do some advanced syncing. The initial authentification works, but during sync, owncloudcmd bails out with "authentification failed". 

The complete log file can be viewed here: https://gist.github.com/felicitus/7e9bcfa06d49899af2df

**Comment by ogoffart** [comment]
I can't see why it does not work :-(
Is the normal owncloud client working fine?
Is there anything special with your setup? (like proxy)

**Comment by felicitus** [comment]
Yes, the regular owncloud client works fine. No proxy, just a plain web server.

**Comment by dragotin** [comment]
there are some strange server replies in the log:
```
02-13 22:16:42:676 void Mirall::AbstractNetworkJob::slotFinished() 5 "Operation canceled" 
```
why is that happening?

**Comment by ckamm** [comment]
@dragotin I think that's the result of the authentication failing. I don't see why this would happen with owncloudcmd and not with the full client. :/

**Comment by felicitus** [comment]
Would it help if you had access to that particular account?

**Comment by ckamm** [comment]
@felicitus Yes, being able to reproduce the issue would help a lot. If the account has no sensitive data, you could email the info to me at: kamm at incasoftware.de.

**Comment by felicitus** [comment]
This is actually bogus. Debian provides the package "owncloud-client-cmd", and the owncloud repo includes owncloudcmd in "owncloud-client", which I wasn't aware of. So in reality, I was running owncloudcmd 1.7.0 when I assumed I was running the latest version.

It would certainly help if owncloudcmd would output it's version number, which isn't the case for the latest version as well. I'll leave this issue open as a reminder to implement version number display to owncloudcmd.

**Comment by ckamm** [comment]
@felicitus There's `owncloudcmd --version`, which should do what you want.

**Comment by felicitus** [comment]
You're right, it didn't seem to be available in 1.7.0.
